pkgs/development/python-modules/torch/default.nix

```diff
@@ -364,6 +364,9 @@ buildPythonPackage rec {
   # NB technical debt: building without NNPACK as workaround for missing `six`
   USE_NNPACK = 0;
 
+  # Explicitly enable MPS for Darwin
+  USE_MPS = setBool stdenv.hostPlatform.isDarwin;
```

Comment:
Maybe at some point we should move these to env, and use `optionalAttrs` for platform-specific stuff, because the way we do it now we trigger a useless rebuild on linux. But that's OK.

Reply:
(just waiting for ofborg-eval now)